### DISCUSSION TITLE
Line chart: Option to display values next to the data points

### DISCUSSION THREAD

**Discussion by leocape** [apache/superset#18447] [Ideas]
**Is your feature request related to a problem? Please describe.**
When looking at a line chart, it's difficult to see what the precise values of the data points are - currently we need to gauge their approximate value by eyeballing the x and y axis, which is straining and not accurate. Especially for quick glances / reporting in meetings.

**Describe the solution you'd like**
To have the values displayed next to the data points themselves, so that we can see the precise values without having to approximate them by eyeballing the axes. This should be optional, and can be shown/hid in the Customize menu

**Describe alternatives you've considered**
This is really the only approach

**Additional context**
&nbsp;
As is:
&nbsp;
![image](https://user-images.githubusercontent.com/67806134/108034365-15259580-703e-11eb-9647-db6fa4f158e8.png)

&nbsp;
To be:
![image](https://user-images.githubusercontent.com/67806134/108034647-7b121d00-703e-11eb-8f29-32eced1ad7b8.png)

&nbsp;
Example of how Looker does it:
&nbsp;
![image](https://user-images.githubusercontent.com/67806134/108034896-e1973b00-703e-11eb-8c8d-37cf79ce9a68.png)
&nbsp;
Where the option to show / hide might be in the UI:
&nbsp;
![image](https://user-images.githubusercontent.com/67806134/108035493-aea17700-703f-11eb-8d0e-f1d3f73c3a14.png)

**Comment by madarapumeghana** [comment]
I want to display the label and value for each data point. When multiple lines are present in the line chart, it is difficult to say which line corresponds to which metric without moving the cursor onto the chart. Is there any option to get both label and values to be displayed on single line chart as shown below.
![LabelValues](https://user-images.githubusercontent.com/115133484/202175861-23dadee2-599a-4094-beae-8d69b5d59bb2.png)
